### DISCUSSION TITLE
Why are our poverty rates so much lower than the actual 2022 SPM?

### DISCUSSION THREAD

**Discussion by MaxGhenis** [PolicyEngine/policyengine-us#3637] [Q&A]
[Our poverty rates in 2022, 2023, and 2024](https://colab.research.google.com/drive/1s7sqyQY_aEBYCSEVcNZihBj458t-2X0X?authuser=1#scrollTo=fK3QMaiJawkv) are broadly about a third lower than the [actual 2022 SPM](https://www.census.gov/content/dam/Census/library/publications/2023/demo/p60-280.pdf#page=17):

Data | Child (%) | Adult (%) | Senior (%) | Overall (%)
-- | -- | -- | -- | --
Actual 2022 | 12.4 | 11.9 | 14.1 | 12.4
PolicyEngine 2022 | 7.0 | 8.2 | 7.6 | 7.8
PolicyEngine 2023 | 7.4 | 9.1 | 9.1 | 8.7
PolicyEngine 2024 | 7.3 | 9.1 | 9.2 | 8.7

We should be calculating taxes and benefits in a generally similar way. We may estimate higher amounts for programs like SSI and WIC that we calibrate to match administrative aggregates, but I don't think this would cause such a large gap.

**Comment by MaxGhenis** [comment]
@nikhilwoodruff and I looked into this and it seems to result from benefit data, especially SNAP which the ASEC understates: in 2022, the [ASEC totals $45 billion, we projected $184 billion](https://colab.research.google.com/drive/1s7sqyQY_aEBYCSEVcNZihBj458t-2X0X?authuser=1#scrollTo=YgAlrLlygMYD), and [actuals were $114 billion](https://fns-prod.azureedge.us/sites/default/files/resource-files/snap-annualsummary-1.pdf). Our 2023 projection is closer: $118 billion vs. $115 billion forecasted.

Beyond survey data inaccuracy, we're over because we don't model asset limits or nonparticipation.

The ASEC is similarly low on SSI: $49 billion vs $70 billion, to which we calibrate.

These gaps seem sufficiently large to explain the gap. On balance our numbers are probably closer to the truth than the ASEC, but we still have room to improve via enhancements already on our roadmap.

**Comment by MaxGhenis** [comment]
Likely due to https://github.com/PolicyEngine/policyengine-us/issues/3686

**Comment by MaxGhenis** [comment]
After merging #3686, which added broadband subsidies, medical expenses, and capped childcare and work expenses to the baseline, and which inflation-indexed some poverty-relevant values including the threshold, the baseline poverty rates rose a bit ([notebook](https://colab.research.google.com/drive/1s7sqyQY_aEBYCSEVcNZihBj458t-2X0X)).


Year | Child Poverty Rate | Working-Age Adult Poverty Rate | Senior Poverty Rate | Overall Poverty Rate
-- | -- | -- | -- | --
2023 | 0.073796 | 0.091509 | 0.089556 | 0.087317
2024 | 0.079544 | 0.097739 | 0.104248 | 0.094999
2025 | 0.084213 | 0.101608 | 0.110538 | 0.099486

**Comment by MaxGhenis** [comment]
Moving @nikhilwoodruff's comment from https://github.com/PolicyEngine/policyengine-us/issues/4146, with regard to the ECPS:

[Notebook 1](https://gist.github.com/nikhilwoodruff/c86bceb1b9b84eab107675e4852d8a41)

[Notebook 2](https://gist.github.com/nikhilwoodruff/8c9ca57455af3ce872d0195d5c3f8803)

Unclear why the child poverty rate is 4.7% in 2022. The values between 2022 and 23 are not substantially different, the targets are mostly similar (no deviations of more than 15%), but the share of weight from PUF records is ~10% in 23 and 24 but ~30% in 22.

**Comment by MaxGhenis** [comment]
Couple ideas:
1. More targets, e.g. population, maybe also by state
2. Remove CPS-based targets https://github.com/PolicyEngine/policyengine-us/issues/4150
3. Slice the year-by-year PUF weight % by subgroup; we've already seen it's a bigger gap for children, what about low-income children, etc?

**Comment by nikhilwoodruff** [comment]
Agree we should get around all of these but they seem very unlikely to reveal the root cause (if there is an intuitive cause) since they are also present in 2023+.

**Comment by MaxGhenis** [comment]
Numbers from the [latest ECPS](https://colab.research.google.com/drive/1s7sqyQY_aEBYCSEVcNZihBj458t-2X0X#scrollTo=fK3QMaiJawkv), which adds filing status calibration:


Year | Child Poverty Rate | Working-Age Adult Poverty Rate | Senior Poverty Rate | Overall Poverty Rate
-- | -- | -- | -- | --
2022 | 0.053537 | 0.166787 | 0.174441 | 0.143950
2023 | 0.093859 | 0.145264 | 0.126783 | 0.130296
2024 | 0.116209 | 0.155968 | 0.098779 | 0.135555
2025 | 0.108113 | 0.149446 | 0.089890 | 0.128118
